source/_components/person.markdown

```diff
@@ -15,10 +15,20 @@ ha_release: 0.88
 
 The person component allows to connect device tracker entities to one or more person entities. The state updates of a connected device trackers will set the state of the person. When multiple device tracers used, the state of berson will be determined next way:
 
-1. If there are sources presenting status 'home', than latest of this sources will be taken.
+1. If there are stationary sources (which type is not 'gps') presenting status 'home', than latest of this sources will be taken.
 2. If there are sources of type 'gps', than latest of this sources will be taken.
 3. Otherwise will be taken latest source with status 'not_home'.
 
+Lets say for example, that you have 3 trackers: 'tracker_gps', 'tracker_router' and 'tracker_ble'.
+
+1. You're at home, all 3 devices shows status 'home' - status of your Person entity will be 'home' with source 'tracker_gps'.
```

Comment:
Won't this case use tracker_router or tracker_ble as source, since those are non gps?

Reply:
Yes, exactly! Thanks, I've corrected this.